### ISSUE TITLE
Unable to build master branch

### ISSUE THREAD

**Issue by gizmotronic** [rancher/os#1992]
**RancherOS Version: master starting with commit 49e4315**

**Where are you running RancherOS? not applicable**

I've been unable to build RancherOS for several weeks but have been otherwise occupied. I finally found the time to use bisect to find the problem, and it appears that this is the culprit:

```
commit 49e4315251e7ea52e1cf9ee5dea8f35aa8eaea1a
Author: Sven Dowideit <SvenDowideit@home.org.au>
Date:   Wed Jun 21 03:29:50 2017 +0000

    Try SSP_ALL glibc buildroot
```

The build proceeds until this point:

```
build-image: Building rancher/os-base:49e4315
DEBU[0000] Running /usr/bin/docker [build -f images/01-base/Dockerfile -t rancher/os-base images/01-base]
Sending build context to Docker daemon 27.65 kB
Step 1/7 : FROM rancher/os-rootfs
 ---> a5773d156549
Step 2/7 : RUN ln -s /dev/null /etc/udev/rules.d/80-net-name-slot.rules
 ---> Running in 1171e435d858
FATAL: kernel too old
The command '/bin/sh -c ln -s /dev/null /etc/udev/rules.d/80-net-name-slot.rules' returned a non-zero code: 127
DEBU[0002] Failed running /usr/bin/docker [build -f images/01-base/Dockerfile -t rancher/os-base images/01-base]: exit status 1
FATA[0002] exit status 1
FATA[0025] exit status 1
make: *** [default] Error 1
```

My build host is running Ubuntu 14.04.5 LTS with the HWE stack kernel, 4.4.0-83-generic. I've also tried Ubuntu 16.04.2 LTS, also with the HWE stack kernel, 4.8.0-58-generic. Both fail with the same error at the same place.

**Comment by SvenDowideit** [comment]
yup, I've basically just upgraded my kernel and kept working on the 1.1.0 debugging - good tracking it down.. :)

**Comment by lachnerd** [comment]
Any solution oin this ?

**Comment by gizmotronic** [comment]
Ubuntu 16.04.3 LTS with the 4.10.0-35-generic HWE kernel is able to compete a build, but cannot run tests successfully.

My build machine is stuck at 14.04 LTS for chromiumos so I'll just have deal with it. It seems reasonable to require the current LTS version.

Is it possible to build using RancherOS? I know that integration tests fail there, too (#1807) but this would be preferable to a dedicated machine.

**Comment by TunedMidja** [comment]
I'm also having this issue and had to downgrade to 1.0.3 as a workaround but would really like to go up to 1.1.0.

**Comment by TunedMidja** [comment]
As @gizmotronic suggested, updating the kernel is also an option. I've successfully built v1.1.0 after having patched the build machine as follows:
sudo apt-get install --install-recommends linux-image-4.10.0-40-generic

**Comment by giper45** [comment]
Hello, news aobout this?

**Comment by niusmallnan** [comment]
If you got the `kernel too old`, you should upgrade your kernel.
Check this kernel version: https://github.com/rancher/os/blob/master/Dockerfile.dapper#L67
Your kernel needs to be higher than this version.

**Comment by muneerk** [comment]
I'm using Master branch rancher os while build with below config, getting error FATAL : KERNEL TOO OLD

https://github.com/rancher/os/blob/master/Dockerfile.dapper#L67

**Comment by bryman55** [comment]
Was able to get past the FATAL: kernel too old.   Even with line L67 changed to higher version you will get FATAL: kernel too old when jumping up to say 4.15+ kernels.  The issue is, you are trying to compile against the existing OLD running kernel.  You need to boot your system up with your NEW compiled kernel and then follow these instructions https://rancher.com/docs/os/v1.x/en/installation/custom-builds/custom-rancheros-iso/ to build your new file system.

I actually just injected the NEW vmlinuz kernel into RancherOS's boot dir, updated syslinux and rebooted.  Got few errors and Waiting for system-docker will run for 300 seconds and timed out.  But everything is fine on boot.  I then just followed the iso link above to created the rest.

Cheers!
